crates/bevy_pbr/src/material.rs

```diff
@@ -3,12 +3,22 @@ use bevy_reflect::TypeUuid;
 use bevy_render::{color::Color, renderer::RenderResources, shader::ShaderDefs, texture::Texture};
 
 /// A material with "standard" properties used in PBR lighting
+/// Standard property values with pictures here https://google.github.io/filament/Material%20Properties.pdf
 #[derive(Debug, RenderResources, ShaderDefs, TypeUuid)]
 #[uuid = "dace545e-4bc6-4595-a79d-c224fc694975"]
 pub struct StandardMaterial {
-    pub albedo: Color,
+    /// Doubles as diffuse albedo for non-metallic, specular for metallic and a mix for everything in between
+    pub base_color_factor: Color,
```

Comment:
`albedo` -> `base_color` was a good call. Blender, Unreal, and Filament use that terminology, whereas Godot and Unity use `albedo`. I think `base_color` is the more approachable name. While `base_color_factor` is the more "descriptive / correct" name, I think `base_color` is nicer to look at / type. I think we should use an `x` and `x_texture` naming convention for these properties.

Reply:
The reason for `_factor` is that this is literally a factor in the final color: `base_color_factor * base_color` where `base_color` is whatever is in the `base_color_texture`. @josh015 actually proposed taking it a step further and using `base_color_range`, where we `mix(base_color_range_min, base_color_range_max, base_color)` because it gives even more control.

I guess I should add some documentation explaining what the property does exactly, either way.

Reply:
Yup I know its a factor (which is why I called it more descriptive / correct). I just think that the majority of people will use one or the other, and `base_color` appeals to my minimalist sensibilities more / seems more newbie friendly. And the people that want to use it as a "factor" will get the behavior they expect.

Reply:
Ok. I'll change it and add a descriptive comment at the same time.

Reply:
Done.

Reply:
> Edit: I didn't mean to be condescending, this was actually new to me as I haven't done much graphics stuff before.
Its all good. I didn't interpret it that way.

Thanks!